app/models/flexible_page/flexible_section/rich_content.rb

```diff
@@ -0,0 +1,19 @@
+module FlexiblePage::FlexibleSection
+  class RichContent < Base
+    ContentImage = Data.define(:alt, :src)
+
+    attr_reader :contents_list, :govspeak, :image
+
+    def initialize(flexible_section_hash, flexible_page)
+      super
+
+      @contents_list = ContentsOutline.new(data["contents_list"])
+      @govspeak = data["govspeak"]
+
+      if data["image"].present?
+        alt, src = data.fetch("image").values_at("alt", "src")
```

Comment:
I forgot to mention in the previous review, this is a nice use of `fetch` with `values_at`.